lib/amazon_ssa_support/evm_bucket.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'evm_common'
+
+module AmazonSsaSupport
+  module EvmBucket
+    
+    def self.get(arg_hash)
+      raise ArgumentError, "evm_bucket must be specified." if arg_hash[:evm_bucket].nil?
+
+      evm_bucket_name  = arg_hash[:evm_bucket]
+      reply_prefix     = arg_hash[:reply_prefix] || AmazonSsaSupport::DEFAULT_REPLY_PREFIX
+      s3               = arg_hash[:s3]  || Aws::S3::Resource.new(region: DEFAULT_REGION)
+      
+      unless (evm_bucket = s3.bucket(evm_bucket_name)).exists?
+        $log.info "#{self.name}.#{__method__}: Reply bucket #{evm_bucket_name} does not exist, creating..."
```

Comment:
I got a comment on another PR today that there should be "(" and ")" around logging calls in all cases.  We aren't going to go back and change existing files but since this is all new code I guess we should add them in all the log calls in this repo.
Also should this and the call below be $log.debug instead?